### ISSUE TITLE
ReferenceError when using flow `import type`

### ISSUE THREAD

**Issue by yusefnapora** [speedskater/babel-plugin-rewire#72]
If a module containing a declaration like `import type {someType} from './someFile';` is imported when `babel-plugin-rewire` is enabled, a `ReferenceError` is thrown:

```
var someType = _someTypeTemp;
                  ^

ReferenceError: _someTypeTemp is not defined
```

Other uses of flow don't have any issues, so for now I can work around this by re-declaring the type inside the module I'm testing.  

This seems like it may be related to #69, but I'm not familiar enough with the internals of babel's flow-stripping transformer or of babel-plugin-rewire to know for sure.

**Comment by speedskater** [comment]
Thanks for reporting this bug and the Delay on this issue but I am currently on holidays and will return on the 23rd of november. I will then fix this bug and create a new beta.

**Comment by speedskater** [comment]
@yusefnapora Your issue should be fixed in the newest beta version 1.0.0-beta-2 for babel 6. Could you all test it and if you have any issues with it could you please provide a PR with a minimal sample reproducing your error (analog to the samples in the repo used by https://github.com/speedskater/babel-plugin-rewire/blob/master/usage-tests/BabelRewirePluginUsageTest.js.
